lang/chicken-scheme/Makefile

```diff
@@ -8,27 +7,38 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=chicken-scheme-interpreter
-PKG_VERSION=5.1.0
+PKG_NAME:=chicken-scheme
+PKG_VERSION=5.2.0
 PKG_RELEASE:=1
 
-PKG_SOURCE:=chicken-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://code.call-cc.org/releases/$(PKG_VERSION)/
-PKG_HASH:=5c1101a8d8faabfd500ad69101e0c7c8bd826c68970f89c270640470e7b84b4b
 PKG_BUILD_DIR:=$(BUILD_DIR)/chicken-$(PKG_VERSION)
+PKG_SOURCE:=chicken-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://code.call-cc.org/releases/5.2.0/
+PKG_HASH:=819149c8ce7303a9b381d3fdc1d5765c5f9ac4dee6f627d1652f47966a8780fa
 
-PKG_MAINTAINER:=Jeronimo Pellegrini <j_p@aleph0.info>
 PKG_LICENSE:=BSD-3-Clause
 PKG_LICENSE_FILES:=LICENSE
 
 include $(INCLUDE_DIR)/package.mk
 
+
+define Package/chicken-scheme/Default
+SECTION:=lang
+CATEGORY:=Languages
+URL:=https://call-cc.org
+MAINTAINER:=Jeronimo Pellegrini <j_p@aleph0.info>
+endef
+
+
+##
+## chicken-scheme-interpreter
+##
+
 define Package/chicken-scheme-interpreter
+$(call Package/chicken-scheme/Default)
+  TITLE:=Chicken Scheme -- interpreter only
   SECTION:=lang
   CATEGORY:=Languages
-  TITLE:=Chicken Scheme
-  URL:=https://call-cc.org
-  ABI_VERSION:=11
```

Comment:
Why?

Reply:
Since now the Makefile produces two packages, I included a meta-package "chicken-scheme". Should I delete that section?

Reply:
I meant the ABI_VERSION section. It's meant to handle package upgrades with packages that depend on this one.